### DISCUSSION TITLE
case-studies/guardians-of-himalayas/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#277] [Website Discussions]
# case-studies/guardians-of-himalayas/

Written by Achu P. Chandra.

https://fcp.cafe/case-studies/guardians-of-himalayas/

**Comment by simplevideoedit** [comment]
Fantastic! Please keep sharing more case studies and timelines. Always great to see

**Comment by florian-duffe** [comment]
Congrats Achu! And thanks for your case study! One question left for me: How could you sync audio from the H6, waveforms? Any chance to automate that?

**Reply by apcdgworks** [reply]
There is 4 audio tracks, I created compound clips and assignee roles and sub roles after that i put that compound clip intro a multi track sequence or synchronised the clip
